lib/operations/create_indexes.js

```diff
@@ -41,21 +62,29 @@ class CreateIndexesOperation extends OperationBase {
       }
     }
 
-    options = Object.assign({}, options, { readPreference: ReadPreference.PRIMARY });
-
-    // Execute the index
-    executeCommand(
-      coll.s.db,
-      {
-        createIndexes: coll.collectionName,
-        indexes: indexSpecs
-      },
-      options,
-      callback
-    );
+    const cmd = {
+      createIndexes: this.collection.collectionName,
+      indexes: indexSpecs
+    };
+
+    if (options.commitQuorum != null) {
+      // Ensure commitQuorum not passed if server < 4.4
+      if (maxWireVersion(server) < 9) {
+        return callback(
+          MongoError.create({
+            message: '`commitQuorum` option for `createIndexes` not supported on servers < 4.4',
+            driver: true
+          })
+        );
+      }
+      cmd.commitQuorum = options.commitQuorum;
+    }
+
+    this.decorateWithCollation = false;
```

Comment:
ah I think I see why you want this now, you don't want it to apply collation to the top-level command. I think it's fine to delete collation on the command in the constructor if it's been passed in. That will ensure the super class doesn't try to apply it. We should evaluate whether collation should be added the way it is in the super class in the future.

Reply:
I did it this way because then the version check for `collation` still gets run by `CommandOperationV2` - if I strip it from the options first, that check doesn't get run. It's really only an issue for `CreateIndexOperation`, I just included it on `CreateIndexesOperation` for consistency, but for the plural command `collation` is specified for each index inside the `indexSpec`, it isn't passed in to the `options`. So if everything is refactored to use the plural operation, this is a non-issue.